components/tenant-mgt/org.wso2.carbon.tenant.dispatcher/src/main/java/org/wso2/carbon/tenant/dispatcher/MultitenantDispatcher.java

```diff
@@ -49,14 +49,20 @@ public AxisService findService(MessageContext mc) throws AxisFault {
             if (to.startsWith(mc.getConfigurationContext().getServiceContextPath())) {
                 int tenantDelimiterIndex = to.indexOf("/t/");
                 if (tenantDelimiterIndex != -1) {
-                    AxisConfiguration ac = mc.getConfigurationContext().getAxisConfiguration();
-                    return ac.getService(MultitenantConstants.MULTITENANT_DISPATCHER_SERVICE);
+                    getService(mc);
                 }
+            } else if (to.startsWith("/t/")) {
```

Comment:
Shall we use a constant defined in a common place and refer it for "/t/"
In lots of places we use it.

Reply:
done